### DISCUSSION TITLE
How do I install CXPatcher for Whisky / DXVK ?

### DISCUSSION THREAD

**Discussion by serista** [Whisky-App/Whisky#196] [General]
Now that Whisky supports also DXVK I'm wondering if I can use CXPatcher as I can with CrossOver, and how? 

In the case of CrossOver I just drag and drop CrossOver over CXPatcher window and it's done.

PS: Actually, incorrectly formulated question because CXPatcher specifically made for patching CrossOver, making DXVK work so much better. But it would be super nice if the same could be done for DXVK used by Whisky.

**Comment by IsaacMarovitz** [comment]
CXPatcher replaces stock CX DXVK with Gcenx's DXVK builds, which includes several patches and fixes. Whisky uses Gcenx's DXVK out of the box. There is no need for CXPatcher.
